frontend/src/component/project/Project/ProjectHealth/ReportTable/ReportCard/ReportCard.tsx

```diff
@@ -166,6 +193,11 @@ export const ReportCard = ({ healthReport }: IReportCardProps) => {
                         <StyledAlignedItem>
                             Review your feature toggles and delete unused
                             toggles.
+                            <br/>
```

Comment:
should be a styled link with margin top. I know it's easier to add br but it's abusing HTML semantics